measurement-tool.html

```diff
@@ -242,6 +243,246 @@
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/measurementToolFolderAdd.jpg" class="align-center" src="images/measurementToolFolderAdd.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">Use the drop-down menus below the pane, to refine the search parameters by selecting <span class="bold">Anywhere in the name</span> or <span class="bold">Start of the name</span>, <span class="bold">All</span>, <span class="bold">Owned by me</span>, or <span class="bold">Owned by others</span> from the menus.</p>
+
+<p>Create a new folder by typing the name in the text box at the bottom of the window, add any description, and clicking <span class="bold">Add</span>.</p>
+
+<p>Click <span class="bold">Save</span> to add the folders to the Manage display.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/measurementToolFolderAddNew.jpg" class="align-center" src="images/measurementToolFolderAddNew.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<li><p class="first">In the <span class="bold">Manager</span> tab of the Measurement Tool, use the <span class="bold">Filter ROI Folders</span> text box to filter the folders displayed.</p>
+
+<p>Folders owned by other users have the shared flag on them.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/measurementToolFolderFilter.jpg" class="align-center" src="images/measurementToolFolderFilter.jpg" style="width: 90%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">Right-click on a ROI and select <span class="bold">Add to Folder</span> to add the ROI to an existing or new folder.</p>
+
+<p>On opening, the <span class="bold">Add to Folder</span> dialog will only show folders currently displayed in the Management Tab.</p>
```

Comment:
You have dialog here and dialogue above.

Also, this is 'Add to ROI Folders' in the screenshot